recipes/mne-python/construct.yaml

```
@@ -51,177 +51,74 @@ menu_packages:
-  - dipy =1.11.0
-  # Time-frequency analysis
-  - pactools =0.3.1
-  - tensorpac =0.6.5
-  - emd =0.8.1
-  - neurodsp =2.3.0
-  - bycycle =1.1.0
-  - fooof =1.1.1
-  # Microstates
-  - mne-microstates =0.3.0
-  - pycrostates =0.6.1
-  # OpenNeuro.org data access
-  - openneuro-py =2024.2.0
-  # sleep staging
-  - sleepecg =0.5.9=*_2
-  - yasa =0.6.5
-  # various biological signals (ECG, EOG, EMG, …)
-  - neurokit2 =0.2.10
-  - mnelab =1.0.0
-  # other
-  - pyriemann =0.8
-  - pyprep =0.4.3=*_1
-  - python-picard =0.8
-  - pybv =0.7.6
-  - edfio =0.4.7  # allow_outdated, for some reason 0.4.8 won't download
-  - eeglabio =0.0.3
-  - mffpy =0.10.0
-  - openmeeg =2.5.15
-  - python-neo =0.14.1
-  - nitime =0.11
-  - snirf =0.8.0
-  - alphacsc =0.4.1
-  # Not on PyPI or no wheels on PyPI
-  # conpy: https://aaltoimaginglanguage.github.io/conpy/
-  # posthoc: https://users.aalto.fi/~vanvlm1/posthoc/python/
-  # eelbrain: https://eelbrain.readthedocs.io/en/stable/index.html
-  # Not on conda-forge
-  # niseq: https://github.com/john-veillette/niseq
-  # sesameeg: https://pybees.github.io/sesameeg
-  # invertmeeg: https://github.com/LukeTheHecker/invert
-  # Meggie: https://github.com/cibr-jyu/meggie
-  # MEM: https://github.com/multifunkim/best-python
-  # <<< END RELATED SOFTWARE LIST >>>
-  # ^^^ this line is special and used by MNE-Python, do not change it!
-
-  - antropy =0.1.9
-  - mayavi =4.8.2=pyside6_*
-  - traitsui =8.0.0
-  - pyface =8.0.0=*_1
-  - imageio-ffmpeg =0.6.0
-  - pandas =2.2.3
-  - polars =1.27.1
+  - mamba =2.1.1
+  # Scientific Python
   - scipy =1.15.2
   - numpy =2.1.3  # allow_outdated, each new version has to wait for numba
   - openblas =0.3.28  # allow_outdated, NumPy etc. need to update
   - libblas =3.9.0=*openblas
+  # Web
+  - requests =2.32.3
+  # Data science and statistics.
+  - pandas =2.2.3
+  - polars =1.29.0
+  - scikit-learn =1.6.1
+  - statsmodels =0.14.4
+  # https://pingouin-stats.org
+  - pingouin =0.5.5
+  # Jupyter
   - jupyter =1.1.1
   - jupyterlab =4.4.1
+  - nbclassic =1.3.1
   - ipykernel =6.29.5
   - spyder-kernels =3.0.3
   # TODO: Needs to not require pyqt
   # https://github.com/spyder-ide/spyder/issues/20201
   # spyder =6.1.0
-  - darkdetect =0.8.0
-  - qdarkstyle =3.2.3
-  - numba =0.61.2
-  - cython =3.0.12
   # I/O
-  - pyxdf =1.17.0
   - openpyxl =3.1.5
   - xlrd =2.0.1
-  # Statistics
-  - pingouin =0.5.5
-  # NeuroSpin needs the following
-  - questionary =2.1.0
-  - pqdm =0.2.0
-  - astropy =7.0.1
+  # https://github.com/Roche/pyreadstat
+  - pyreadstat =1.2.8
+  # Image processing
+  - scikit-image =0.25.2
+  - pillow =11.2.1
+  # Data
+  - pooch =1.8.2
+  # Symbolic math
+  - sympy =1.14.0
   # Viz
   # matplotilb is just matplotlib-base, tornado, and pyqt
   # https://github.com/conda-forge/matplotlib-feedstock/blob/main/recipe/meta.yaml
-  - matplotlib-base =3.10.1
+  - matplotlib-base =3.10.3
   - tornado =6.4.2
   - pyside6 =6.8.2  # allow_outdated, PyVista (VTK 9.3.1) and Mayavi
```

Both VTK and Mayavi were removed as dependencies. 

```suggestion
  - pyside6 =6.8.2  
```

---

this probably means that we can/should bump the pin?  I'm guessing that the comment is used in tests that check that the pins reflect the most current available version, so if we simply remove it without bumping the pin, it will cause a test failure.